prody/proteins/pdbfile.py

```diff
@@ -529,6 +529,10 @@ def _parsePDBLines(atomgroup, lines, split, model, chain, subset,
                 hetero[acount] = startswith[0] == 'H'
                 segnames[acount] = line[72:76]
                 elements[acount] = line[76:78]
+                try:
+                    charges[acount] = int(line[79] + line[78])
```

Comment:
why is it 79 then plus 78?

Reply:
Because the sign is second in the PDB for some reason: http://www.wwpdb.org/documentation/file-format-content/format33/sect9.html#ATOM
"Columns 79 - 80 indicate any charge on the atom, e.g., 2+, 1-. In most cases, these are blank."

Reply:
Ok sounds good. Just needed to make sure!